services/api/package.json

```diff
@@ -33,13 +33,13 @@
   },
   "dependencies": {
     "@apollo/server": "4.9.1",
+    "@axe-core/webdriverjs": "^4.7.3",
```

Comment:
Without selenium we are not using a webdriver anymore so we shouldn't need that library.